package.json

```diff
@@ -17,6 +17,7 @@
   "scripts": {
     "build": "make build",
     "build-docs": "cd ./www && \"$npm_execpath\" install && \"$npm_execpath\" run build",
+    "build-docs:with-theme": "cd ./www && \"$npm_execpath\" install && \"$npm_execpath\" run build:with-theme",
```

Comment:
If this is working correctly, should we remove the `(coming soon)` comment in earlier documentation?

Reply:
Currently the theme to be used is hard-coded to use the edx theme baked into the project: https://github.com/edx/paragon/pull/573/files#diff-fb043578f52339bcb2a2d2b4df1203c8R7-R10

This will change with the creation of the @edx/brand-edx.org repository.